packages/commonwealth/client/scripts/models/IChainAdapter.ts

```diff
@@ -44,37 +40,20 @@ abstract class IChainAdapter<C extends Coin, A extends Account> {
   public async initServer(): Promise<boolean> {
     clearLocalStorage();
     console.log(`Starting ${this.meta.name}`);
-    const [response] = await Promise.all([
-      axios.get(`${SERVER_URL}/bulkOffchain`, {
-        params: {
-          chain: this.id,
-          community: null,
-          jwt: userStore.getState().jwt,
-        },
-      }),
-    ]);
 
+    // TODO: Do we really need this for 1inch only? -- ask product.
```

Comment:
as far as I remember it was a request for 1inch as they wanted to have dark mode by default. It is done by not elegant way though. Worth to double check